gallery/src/components/NxTable/NxTablePage.tsx

```diff
@@ -48,486 +90,660 @@ const paginationCodeExamples = [
+            paging.
+          </GalleryExampleTile>
+
+          <GalleryExampleTile title="Pagination and Filtering Example"
+                              id="nx-table-pagination-filter-example"
+                              liveExample={NxTablePaginationFilterExample}
+                              codeExamples={paginationFilterCodeExamples}
+                              collapseCodeExample>
+            An example of a table with an <NxCode>NxPagination</NxCode> component in the footer to control
+            paging as well as a row of filter headers. Demonstrates the use of
+            the <NxCode>pagination-table-height</NxCode> SCSS function when a filter row is present.
+          </GalleryExampleTile>
+
+          <GalleryExampleTile title="Loading Example"
+                              id="nx-table-loading-example"
+                              liveExample={NxTableLoadingExample}
+                              codeExamples={tableLoadingExample}
+                              collapseCodeExample>
+            An example of how <NxCode>NxTable</NxCode> should be used while its data is loading.
+          </GalleryExampleTile>
+
+          <GalleryExampleTile title="Error Example"
+                              id="nx-table-error-example"
+                              liveExample={NxTableErrorExample}
+                              codeExamples={tableErrorExample}
+                              collapseCodeExample>
+            An example of how <NxCode>NxTable</NxCode> should be used to indicate that there was an error
+            loading its data.
+          </GalleryExampleTile>
+
+          <GalleryExampleTile title="Empty Example"
+                              liveExample={NxTableEmptyExample}
+                              codeExamples={tableEmptyExample}
+                              collapseCodeExample>
+            An example of how <NxCode>NxTable</NxCode> should be used to indicate that there is no data
+            to be seen.
+          </GalleryExampleTile>
+
+          <GalleryExampleTile title="Custom Meta-Info Example"
+                              liveExample={NxTableMetaInfoExample}
+                              codeExamples={tableMetaInfoExample}
+                              collapseCodeExample>
+            An example of how <NxCode>NxTable</NxCode> should be used with a custom meta-info situation.
+          </GalleryExampleTile>
+        </NxTabPanel>
+        <NxTabPanel>
+          <GalleryExampleTile title="NX Table Simple Example"
+                              htmlExample={NxTableSimpleCode}
+                              codeExamples={NxTableSimpleCode}
+                              collapseCodeExample>
+            A simple, static demonstration of <NxCode>nx-table</NxCode> styles.
+          </GalleryExampleTile>
+
+          <GalleryExampleTile title="NX Table Truncation and Wrapping Example"
```

Comment:
I feel like this doesn't entirely address the fact that different aspects of the styling are shown in the HTML examples than in the React examples, which isn't always obvious to users.  Consider this example here: it shows how truncation and text wrapping work within a table, which is important information whether the table is constructed in react or plain HTML.  Does the reorganization make that any easier to find than it was previously?  I'm not sure if we need a naming adjustment to the tabs, or maybe another tab, but I feel like we need a way to make it clearer to both React users and HTML users where they should look for "styling variations"

Reply:
> I feel like this doesn't entirely address the fact that different aspects of the styling are shown in the HTML examples than in the React examples....

Take a look at this branch: https://github.com/sonatype/sonatype-react-shared-components/tree/RSC-842-SPIKE-tabs--NxTable-variations

I added a "Variations" tab for the extra layouts, and then consolidated the examples which had both React and HTML code.

Reply:
> I added a "Variations" tab for the extra layouts, and then consolidated the examples which had both React and HTML code.

Hmm that means that for examples that have two implementations shown, only one implementation is actually driving the live example, correct? I worry about the other example being incorrect or getting out of sync.

Reply:
> > I added a "Variations" tab for the extra layouts, and then consolidated the examples which had both React and HTML code.
> 
> Hmm that means that for examples that have two implementations shown, only one implementation is actually driving the live example, correct? I worry about the other example being incorrect or getting out of sync.

I think that if we default to showing the React example then the risk is quite low. The React examples still use the same HTML/CSS behind the scenes as the solely HTML/CSS examples.

Reply:
> I think that if we default to showing the React example then the risk is quite low.

I have to disagree, I think that the risk is quite high.  The HTML examples in this scenario are going completely untested.

> The React examples still use the same HTML/CSS behind the scenes as the solely HTML/CSS examples.

This assumption is where the risk lies.  We need a way to verify that it remains true

Reply:
> I have to disagree, I think that the risk is quite high. The HTML examples in this scenario are going completely untested.

In almost all cases they are going completely untested now. If you look at the visual tests we are testing the React page for `NxTable`, but not the HTML examples. This seems to be true but reversed for `nx-list.

Reply:
> In fact they are going completely untested now. If you look at the visual tests we are testing the React page for NxTable, but not the HTML examples. This seems to be true but reversed for nx-list.

That's partially true.  The `nx-table` visual tests include certain examples from both the `NxTable` page and the `nx-table` page (as well as `nx-table-container.  But even the ones that aren't getting automated checks are at least visible on the page and get manually tested.

An idea that I have to improve this is that when we want to show the same thing in both react and plain HTML examples, we could have a tabbed interface or something that switches not just which example code is shown but also which live example is shown - so there's still a way to see the result of the plain HTML example.

That said, another idea is to just minimize the number of plain HTML examples. Have one basic one and then rely exclusively on React examples to show all the different options.  The only issue I see with this is when the React and HTML versions have distinct ways of doing the same thing.  For instance, the different button variants are activated using classes when you're in plain HTML, but have a special `variant` prop when you're in React.

Reply:
> Hmm that means that for examples that have two implementations shown, only one implementation is actually driving the live example, correct? I worry about the other example being incorrect or getting out of sync.

Another way to do this would be to have tabs for the examples as well. We don't have very many components that have both HTML & React examples: Tables, Lists, Alerts, and Buttons the last two which I worry could be out of date.

Reply:
> That said, another idea is to just minimize the number of plain HTML examples.

I don't see how this is much different than what's there now except that we aren't showing the code. There's still no testing to see if there is any divergence.

Reply:
> An idea that I have to improve this is that when we want to show the same thing in both react and plain HTML examples, we could have a tabbed interface or something that switches not just which example code is shown but also which live example is shown - so there's still a way to see the result of the plain HTML example.

Yeah we're on the same wavelength here. I'm not sure if the example code we have now will support this.

Reply:
> But even the ones that aren't getting automated checks are at least visible on the page and get manually tested.

As you said above, "The assumption is where the risk lies." When was the last time anybody compared the `nx-alert` page to the `NxAlert` page?